.github/ISSUE_TEMPLATE/route-shield-request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Route shield request
```

Comment:
It sounds like the plain Markdown file approach is [deprecated](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository) in favor of issue template forms. Should we try those instead?

Reply:
I went to Insights > Community Standards and clicked the Add button next to issue templates. This is what it gave me :shrug: